content/articles/manage-a-record.markdown

```diff
@@ -16,7 +16,7 @@ categories:
 
 You can manage [A records](/articles/a-record) in DNSimple using the [DNS record editor](/articles/record-editor).
 
-The instructions in this article assumes you are familiar with the [A record format](/articles/a-record#record-format) and usage.
+The instructions in this article assume you are familiar with the [A record format](/articles/a-record#record-format) and usage.
```

Comment:
@itsalyse all the 5 `manage-*-record` articles have the same structure. Should this change be reflected in all the other 4 articles?

Reply:
It should. I'm going through and making it all uniform.